### ISSUE TITLE
Chart does not render inside ResponsiveContainer when imported from another file

### ISSUE THREAD

**Issue by enszrlu** [recharts/recharts#4763]
## Do you want to request a feature or report a bug?
Bug

## What is the current behavior?
Chart does not render inside ResponsiveContainer. 

**Does Not Work:**
```
//BarChart passed down as Graph

function Card({ Graph, Title, DataTitle, Labels }: DataCardProps) {
return (
<div className="flex w-full min-w-fit gap-4">
<ResponsiveContainer width="100%" height={100} minWidth={150}>
      <Graph />
</ResponsiveContainer>
</div>)}
```

**Works:** 
```
//BarChart inside ResponsiveContainer passed down as Graph

function Card({ Graph, Title, DataTitle, Labels }: DataCardProps) {
return (
<div className="flex w-full min-w-fit gap-4">
      <Graph />
</div>)}
```

## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via codesandbox or similar (template: https://codesandbox.io/p/sandbox/simple-line-chart-v25dl4).

https://codesandbox.io/p/sandbox/clever-golick-6mpj5c

## What is the expected behavior?

Rendering the chart

## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?

    "recharts": "^2.12.7",

Chrome / MacOs

**Comment by ckifer** [comment]
This is known, will be fixed in 3.0. 

Thanks

**Comment by ckifer** [comment]
Going to close this as its tracked elsewhere https://github.com/recharts/recharts/issues/2788
